torch/csrc/jit/codegen/cuda/fusion_segmenter.h

```diff
@@ -412,12 +441,20 @@ class TORCH_CUDA_CU_API SegmentCandidateFinder {
     return segmented_fusion_->edges();
   }
 
-  Fusion& completeFusion() {
+  Fusion* completeFusion() {
```

Comment:
Originally just because I switched to a `unique_ptr` to own the fusion, which makes pointer return value a better fit. At this point I guess there is not much to this change besides unifying the usage interfaces slightly.